cadastre/server/cadastre_service.py

```diff
@@ -200,11 +200,17 @@ def create_pdf(self, params: Dict[str, str], response: QgsServerResponse, projec
             res.connector
         )
         pmulti = 1
+        all_cities = params.get('ALLCITIES', 'f').lower()
+        if all_cities in ['t', 'true', '1']:
+            all_cities = True
+        else:
+            all_cities = False
```

Comment:
```suggestion
        all_cities = (params.get('ALLCITIES', 'f').lower() in ['t', 'true', '1'])
```
@Gustry like this ?

Reply:
Je ne sais même pas pourquoi tu mets des `()` 😱

```python
all_cities = params.get('ALLCITIES', 'f').lower()
all_cities = all_cities in ['t', 'true', '1']
```
Ca retourne déjà un booléen.

Je viens de pousser, en une ligne
A ce moment la, on passe aussi à `format`
https://realpython.com/python-string-formatting/#1-old-style-string-formatting-operator